packages/core/src/plugins/use-masked-errors.ts

```diff
@@ -6,39 +6,64 @@ export const DEFAULT_ERROR_MESSAGE = 'Unexpected error.';
 export type MaskErrorFn = (error: unknown, message: string) => Error;
 
 export type SerializableGraphQLErrorLike = Error & {
-  name: 'GraphQLError';
-  toJSON(): { message: string };
+  toJSON?(): { message: string };
```

Comment:
Wait I just realized this. Why we remove `name` here? GraphQLError should have `GraphQLError` name as in the following `isGraphQLError` check